doc/API-Mastodon.md

```diff
@@ -29,7 +29,91 @@ For supported apps please have a look at the [FAQ](help/FAQ#clients)
 
 ## Entities
 
-These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/entities/).
+These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/entities/). With some additional extensions listed below.
```

Comment:
Please have only one sentence per line. It will display the same per Markdown syntax and it’s cleaner in line-based versioning software like git.

Reply:
```suggestion
These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/entities/).
With some additional extensions listed below.
```